webapp/src/layouts/Dashboard/index.js

```diff
@@ -1,15 +1,15 @@
 import React, { useState } from 'react'
 import PropTypes from 'prop-types'
 import Hidden from '@mui/material/Hidden'
-import Box from '@mui/material/Box'
 import { makeStyles } from '@mui/styles'
 
 import Sidebar from '../../components/Sidebar'
 import Header from '../../components/Header'
-import Footer from '../../components/Footer'
 import Message from '../../components/Message'
+import Footer from '../../components/Footer'
 
 import styles from './styles'
+import { Toolbar } from '@mui/material'

```

Comment:
please, import it in top of this file.

import order:
1. Third-party libraries
2. Project import
3. Folder import